test/Staking.ts

```diff
@@ -1234,6 +1238,41 @@ describe("Staking", () => {
         await setBalance(await ethers.resolveAddress(staking), ethers.parseEther("2"));
         await status.connect(node.wallet).alive();
     });
+    it("should not cause alive() to PANIC", async() => {
+        const {nodesData, staking, status} = await registeredOnlyNodes();
+        const [node,] = nodesData;
+        const stakingReward = ethers.parseEther("1");
+        const [user1,] = await ethers.getSigners();
+        const walletReward = 10n**14n;
+        const grantRewards = async (node: BigNumberish) => {
```

Comment:
Good function.
Please split it on 2: for particular node and for all enabled nodes and put as external function to `fixtures.ts` near `sendHeartbeat` and others to allow it's further reusing.